### PR TITLE
BIOWDL-380: Update centrifuge task.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md

### CODE REVIEW

File: centrifuge.wdl [hunk 1]
```diff
@@ -1,124 +1,201 @@
 version 1.0
 
-# Copyright Sequencing Analysis Support Core - Leiden University Medical Center 2018
+# Copyright (c) 2020 Sequencing Analysis Support Core - Leiden University Medical Center
```
Comment (DavyCats) [inline centrifuge.wdl]: This file has existed since 2018, so the copyright should read 2018

File: centrifuge.wdl [hunk 2]
```diff
@@ -1,124 +1,201 @@
...
+        String indexBasename = "NAME"
+        String outputPrefix
+
+        Int? offrate
+        Int? ftabChars
         Int? kmerCount
+        File? sizeTable
 
-        Int threads = 8
+        Int cores = 5
```
Comment (DavyCats) [inline centrifuge.wdl]: `threads` matches the option of the tool, so why rename it?

File: centrifuge.wdl [hunk 3]
```diff
@@ -1,124 +1,201 @@
...
-        Boolean noRef = false
-        Boolean justRef = false
-        Int? offRate
-        Int? fTabChars
-        File? nameTable
-        File? sizeTable
-        Int? seed
+        File nameTable
+        File referenceFile
+        String indexBasename = "NAME"
```
Comment (DavyCats) [inline centrifuge.wdl]:
```suggestion
        String indexBasename = "centrifuge_index"
```

File: centrifuge.wdl [hunk 4]
```diff
@@ -1,124 +1,201 @@
...
+        Array[File]? read2
+        Int? trim5
+        Int? trim3
+        Int? reportMaxDistinct
+        String? hostTaxIDs
+        String? excludeTaxIDs
 
-        Int threads = 4
-        String memory = "8G"
+        Int cores = 4
```
Comment (DavyCats) [inline centrifuge.wdl]: Same comment as before, why rename this?

File: centrifuge.wdl [hunk 5]
```diff
@@ -1,124 +1,201 @@
...
+
+        Int? offrate
+        Int? ftabChars
         Int? kmerCount
+        File? sizeTable
 
-        Int threads = 8
+        Int cores = 5
         String memory = "20G"
+        String dockerImage = "quay.io/biocontainers/centrifuge:1.0.4_beta--he860b03_3"
```
Comment (DavyCats) [inline centrifuge.wdl]: Should we be using a beta version?

File: centrifuge.wdl [hunk 6]
```diff
@@ -1,124 +1,201 @@
...
+
+task Inspect {
+    input {
+        String printOption = "fasta"
+        String indexBasename
+        String outputPrefix
+
+        Int? across
+
+        Int cores = 1
```
Comment (DavyCats) [inline centrifuge.wdl]: This input isn't getting used.

File: centrifuge.wdl (hunk 2 above)
Comment (JasperBoom) [reply centrifuge.wdl]: Are we naming these based on tools? I tried to make this specific setting the same in everything BioWDL related I have written. For continuities sake.

File: centrifuge.wdl (hunk 5 above)
Comment (JasperBoom) [reply centrifuge.wdl]: The non beta tagged version on quay.io is from may 2018. So I thought using the most recent one would be better.

File: centrifuge.wdl (hunk 6 above)
Comment (JasperBoom) [reply centrifuge.wdl]: It is by the runtime right? Or can users set that without this input?

File: centrifuge.wdl (hunk 2 above)
Comment (DavyCats) [reply centrifuge.wdl]: Input names are generally based on the tool options/flags they represent

File: centrifuge.wdl (hunk 5 above)
Comment (DavyCats) [reply centrifuge.wdl]: Okay, I guess that's fine.

File: centrifuge.wdl (hunk 6 above)
Comment (DavyCats) [reply centrifuge.wdl]: But it isn't used by the command, so the tool will always just use one thread. There is no point in setting a (configurable) number of  cores in the runtime section.

**Pull request merged**